detail-editace.php

```diff
@@ -0,0 +1,15 @@
+<?php
+require_once "hlavicka.php";
+
+if (!$uzivatelPrihlaseny) {
+    require_once "prihlaseni-formular.php";
+} else {
+?>
+
+TODO: formulář na úpravu članku
```

Comment:
Ony neznají "TODO" ;)

Reply:
to je TODO pro me, abych to jeste dodelal :D